realm-transformer/src/main/groovy/io/realm/transformer/RealmTransformer.groovy

```diff
@@ -278,4 +289,23 @@ class RealmTransformer extends Transform {
         return merged;
     }
 
+    // Use reflection to get the AndroidBuild from Android plugin for getting the path to android.jar
+    // There is no official way to get the path to android.jar for transform.
+    // See https://code.google.com/p/android/issues/detail?id=209426
+    private void addBootClassesToClassPool(ClassPool classPool) {
+        try {
+            def androidBuilderField = BasePlugin.getDeclaredField("androidBuilder");
```

Comment:
This feels so dedelicate... Google could brake this at any release. Also we would need to test it works with a wide range of Android Gradle plugin versions